### ISSUE TITLE
WCAG 3 Draft Review: Thoughts on Design Principles

### ISSUE THREAD

**Issue by melissaegg** [w3c/silver#618]
I'm writing to respond to this question posted on the editor's blog: Are there additional Design Principles and Requirements that should be included in the WCAG 3 project?

As well as these goals put forth for WCAG 3:

Goals for WCAG 3 include:
- be easier to understand
- cover more user needs, including more needs of people with cognitive disabilities

Please consider including design principles related to trauma-informed care in web design. 

A major symptom of trauma is cognitive impairment according to the CDC (the effects of trauma have been well-researched - [see CDC website here](https://www.cdc.gov/violenceprevention/aces/resources.html). For example, a violent childhood may continue to have long-lasting effects years later. 

Many websites, even those offering assistance to people who they know have experienced traumatic situations, do not take into consideration the fundamental principles of trauma-informed care. This leads to people giving up when they are seeking help. The mental health website for veterans with content written at a post-graduate grade level may not actually be helpful at all to the intended audience. 

This is also an equity issue. Note the following
- PTSD is more common in “U.S. Latinos, African Americans, and American Indians” [VA PTSD study](https://www.ptsd.va.gov/publications/rq_docs/V31N4.pdf) and [Child Trends study](https://www.childtrends.org/publications/prevalence-adverse-childhood-experiences-nationally-state-race-ethnicity)
- Low-income young adults are more likely to have exposure to trauma [Public Health study](https://pubmed.ncbi.nlm.nih.gov/33525425/#:~:text=Low%2Dincome%20young%20adults%20are,likely%20to%20have%20sleep%20problems)

- Trauma interferes with education and disproportionately targets students of color, students with disabilities, and those living in poverty [MADC paper](https://maec.org/wp-content/uploads/2019/06/Exploring-Equity-Trauma-Informed-Classrooms-Primer.pdf)

I'm happy to help in this area (background in user experience and trauma-informed design.) Those of us working in tech inadvertently harm people who have already been through traumatic experiences by aggravating the trauma with our lack of care. We can be more inclusive and accessible by following trauma-informed care principles in our tech work.

**Comment by lseeman** [comment]
Hi

Have you seen the work of COGA (the cognitive accessibility task fource? 
Such as [Making Content Usable for People with Cognitive and Learning disabilities - W3C](https://www.w3.org/TR/coga-usable/)? Also see our wiki at https://www.w3.org/WAI/GL/task-forces/coga/wiki/Main_Page

COGA is now working on mental health. Be In Touch if you would like to be involved. I can let you know how to join the task fource

**Comment by melissaegg** [comment]
@lseeman I think this is why my friend told me to get in touch! I have not seen this before though - terrific. I would like to learn more about being involved please. Thank you!

**Comment by rachaelbradley** [comment]
Proposed draft response: The requirements document currently includes criteria on [broad disability support](https://w3c.github.io/silver/requirements/#broad-disability-support). Part of that goal is to better support people with cognitive and learning disabilities. The best way to incorporate more guidance in that area is to work with the COGA task force or community group. Do you have the information you need to connect with them?

**Comment by melissaegg** [comment]
@rachaelbradley yes, I can see people I can email when I have space to contribute! thank you all for putting this on my radar.
